packages/system/suc-upgrade/suc-upgrade.sh

```diff
@@ -8,18 +8,18 @@ echo "SUC_VERSION: $SUC_VERSION"
 if [ "$FORCE" != "true" ]; then
     if [ -f "/etc/kairos-release" ]; then
       # shellcheck disable=SC1091
-      UPDATE_VERSION=$(source /etc/kairos-release && echo "${KAIROS_VERSION}")
+      UPDATE_VERSION=$(source /etc/kairos-release && echo "KAIROS_VERSION=${KAIROS_VERSION};KAIROS_SOFTWARE_VERSION=${KAIROS_SOFTWARE_VERSION}")
     else
       # shellcheck disable=SC1091
-      UPDATE_VERSION=$(source /etc/os-release && echo "${KAIROS_VERSION}")
+      UPDATE_VERSION=$(source /etc/os-release && echo "KAIROS_VERSION=${KAIROS_VERSION};KAIROS_SOFTWARE_VERSION=${KAIROS_SOFTWARE_VERSION}"")
```

Comment:
```suggestion
      UPDATE_VERSION=$(source /etc/os-release && echo "KAIROS_VERSION=${KAIROS_VERSION};KAIROS_SOFTWARE_VERSION=${KAIROS_SOFTWARE_VERSION}")
```

I think there is an extra quote here?